core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -561,7 +561,12 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
       val consumerRebalanceListener = new InternalRebalanceListenerForNewConsumer(this, customRebalanceListener)
       if (whitelistOpt.isDefined) {
         try {
-          consumer.subscribe(Pattern.compile(whitelistOpt.get), consumerRebalanceListener)
+          val revisedWhitelist = whitelistOpt.get.trim
+            .replace(',', '|')
+            .replace(" ", "")
+            .replaceAll("""^["']+""","")
+            .replaceAll("""["']+$""","") // property files may bring quotes
+          consumer.subscribe(Pattern.compile(revisedWhitelist), consumerRebalanceListener)
```

Comment:
Thanks for explaining. It makes sense to me.
On a separate note, do you think it makes sense to add some unit test for this change?